pilot/cmd/pilot-agent/wait.go

```diff
@@ -0,0 +1,86 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"istio.io/pkg/log"
+)
+
+var (
+	timeoutSeconds       int
+	requestTimeoutMillis int
+	periodMillis         int
+	url                  string
+
+	waitCmd = &cobra.Command{
+		Use:   "wait",
+		Short: "Waits until the Envoy proxy is ready",
```

Comment:
@howardjohn Do you have a suggestion for the description here? I guess this command would wait for the Istio-wrapped envoy proxy to be ready, yes?